### ISSUE TITLE
Window size and number of insert mode completions in Neovim

### ISSUE THREAD

**Issue by Markus00000** [junegunn/fzf.vim#214]
- Category
    - [ ] Question
    - [ ] Bug
    - [X] Suggestion
- OS
    - [X] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Etc.
- Vim
    - [ ] Vim
    - [X] Neovim

In small terminal windows insert mode completions show very few lines. For example, a terminal window having 12 lines will only show 2 completions while showing 5 lines of the edited file. In this case I would prefer to see just the current line of the edited file and 6 lines of completions. 6 instead of 2 completions make it easier to select one, especially when using `ctrl-n/p`.

Even using the whole window as in Vim would be preferable to me but this seems to only work for `:FZF` but not for functions like `fzf-complete-path`.

**Comment by junegunn** [comment]
Good point. I think fzf should automatically switch to using tab instead of split if the window is too small. Will look into it.
